src/main/java/org/apache/maven/plugins/enforcer/RequireEncoding.java

```diff
@@ -25,9 +31,7 @@
  * @see <a href="https://github.com/mikedon/encoding-enforcer">mikedon/encoding-enforcer</a>
  * @see <a href="https://github.com/ericbn/encoding-enforcer">ericbn/encoding-enforcer</a>
  */
-public class RequireEncoding
-    implements EnforcerRule
-{
```

Comment:
Please do not reformat code that covers the real changes you are making...and makes it hard to understand the patch..